infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/send/logreader/BaseLogEntryReader.java

```diff
@@ -235,7 +238,7 @@ public LogReplication.LogReplicationEntryMsg read(UUID logEntryRequestId) throws
             return txMessage;
 
         } catch (Exception e) {
-            log.warn("Caught an exception while reading transaction stream {}", e);
+            log.warn("Caught an exception while reading transaction stream", e);
```

Comment:
why to remove placeholder for printing exception?

Reply:
for WARN level log exception doesn't need a placeholder